lib/rubocop/cop/style/access_modifier_declarations.rb

```diff
@@ -235,15 +235,18 @@ def allow_modifiers_on_alias_method?(node)
           cop_config['AllowModifiersOnAliasMethod'] && access_modifier_with_alias_method?(node)
         end
 
+        # rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
         def offense?(node)
           if group_style?
+            return false if !node.parent && access_modifier_with_symbol?(node)
             return false if node.parent&.if_type?
```

Comment:
By doing it this way, the new `rubocop:disable` annotation will likely no longer be necessary.

```suggestion
            return false if node.parent ? node.parent.if_type? : access_modifier_with_symbol?(node)
```

Reply:
Yep, it works. Thanks.